saleor/graphql/order/mutations/fulfillment_update_tracking.py

```diff
@@ -63,7 +71,9 @@ def perform_mutation(  # type: ignore[override]
         fulfillment_tracking_updated(fulfillment, user, app, tracking_number, manager)
 
         notify_customer = input.get("notify_customer")
+        cls.call_event(manager.fulfillment_updated, fulfillment, notify_customer)
         if notify_customer:
+            # this event will be deprecated. Fulfillment_updated will replace it.
             send_fulfillment_update(order, fulfillment, manager)
```

Comment:
You can also mention `NOTIFY` event in `webhook_events_info`. Some of the modules didn't receive `webhook_events_info` yet, but it would be much more easier to assume, that each mutation that already has `webhook_events_info` covers all events.

Reply:
sure, will add missing events if I see they are missing